active-directory-wpf-msgraph-v2/MainWindow.xaml.cs

```diff
@@ -27,6 +27,11 @@ public MainWindow()
             InitializeComponent();
         }
 
+        private void UseWam_Changed(object sender, RoutedEventArgs e)
+        {
+            App.CreateApplication(howToSignIn.SelectedIndex != 2);
```

Comment:
>howToSignIn.SelectedIndex != 2 [](start = 34, length = 30)

should this be comparing to a tag value instead making the code more readable?

Reply:
Actually the tag as I wrote it didn't work (it needs a nested element, and expressing string). I removed the tags from the XAML file